packages/browser-sdk/src/client.ts

```diff
@@ -101,8 +101,8 @@ export class BucketClient {
     this.logger =
       opts?.logger ?? loggerWithPrefix(quietConsoleLogger, "[Bucket]");
     this.context = {
-      user: opts?.user,
-      company: opts?.company,
+      user: opts?.user?.id ? opts.user : undefined,
```

Comment:
I guess maybe a test would be good to have for this scenario. I find that these SDKs require a lot of tests due to the nature of js. Same for the node part.

Reply:
good point